en/docs/setup/configuring-the-datasources.md

```diff
@@ -9,74 +9,20 @@
-                        <maxWait>60000</maxWait>
-                        <testOnBorrow>true</testOnBorrow>
-                        <validationQuery>SELECT 1</validationQuery>
-                        <validationInterval>30000</validationInterval>
-                        <defaultAutoCommit>false</defaultAutoCommit>
-                    </configuration>
-                </definition>
-            </datasource>
-            <datasource>
-                <name>WSO2UserStore</name>
-                <description>The User Store</description>
-                <jndiConfig>
-                    <name>jdbc/WSO2UserStore</name>
-                </jndiConfig>
-                <definition type="RDBMS">
-                    <configuration>
-                        <url>jdbc:mysql://wso2is-pattern1-mysql-service:3306/WSO2UMDB?autoReconnect=true&amp;useSSL=false</url>
-                        <username>wso2carbon</username>
-                        <password>wso2carbon</password>
-                        <driverClassName>com.mysql.jdbc.Driver</driverClassName>
-                        <maxActive>80</maxActive>
-                        <maxWait>60000</maxWait>
-                        <minIdle>5</minIdle>
-                        <testOnBorrow>true</testOnBorrow>
-                        <validationQuery>SELECT 1</validationQuery>
-                        <validationInterval>30000</validationInterval>
-                        <defaultAutoCommit>false</defaultAutoCommit>
-                    </configuration>
-                </definition>
-            </datasource>
-            <datasource>
-                <name>WSO2_IDENTITY_DB</name>
-                <description>The datasource used for registry, user management and identity</description>
-                <jndiConfig>
-                    <name>jdbc/WSO2IdentityDS</name>
-                </jndiConfig>
-                <definition type="RDBMS">
-                    <configuration>
-                        <url>jdbc:mysql://wso2is-pattern1-mysql-service:3306/WSO2_IDENTITY_DB?autoReconnect=true&amp;useSSL=false</url>
-                        <username>wso2carbon</username>
-                        <password>wso2carbon</password>
-                        <driverClassName>com.mysql.jdbc.Driver</driverClassName>
-                        <maxActive>80</maxActive>
-                        <maxWait>60000</maxWait>
-                        <minIdle>5</minIdle>
-                        <testOnBorrow>true</testOnBorrow>
-                        <validationQuery>SELECT 1</validationQuery>
-                        <validationInterval>30000</validationInterval>
-                        <defaultAutoCommit>false</defaultAutoCommit>
-                    </configuration>
-                </definition>
-            </datasource>
-        </datasources>
+        ```
+        [[datasource]]
+           id = "WSO2_CARBON_DB"
+           url = "jdbc:h2:./repository/database/WSO2CARBON_DB;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=60000"
```

Comment:
We no longer recommend to change CARBON DB nor document anything about that. Instead, provide the configuration to  configure Identity and shared databases